src/lib/database.js

```diff
@@ -201,13 +313,64 @@ function buildQuery(parameters) {
   }
 
   return {
-    query: {
-      bool: {
-        must_not: mustNotQueries,
-        filter: filterQueries
-      }
+    bool: {
+      filter: filterQueries,
+    }
+  }
+}
+
+function buildQuery(parameters) {
```

Comment:
maybe rename `buildOpenSearchQuery` to make it clear what query it's building

Reply:
Yeah. Done.